.github/scripts/trymerge.py

```diff
@@ -849,7 +849,7 @@ def merge_ghstack_into(self, repo: GitRepo, force: bool, comment_id: Optional[in
                     continue
                 commit_msg = pr.gen_commit_message(filter_ghstack=True)
                 # Raises exception if matching rule is not found
-                find_matching_merge_rule(pr, repo, force=force, skip_internal_checks=can_skip_internal_checks(self, comment_id))
+                find_matching_merge_rule(pr, repo, force=force, skip_internal_checks=can_skip_internal_checks(self, comment_id), land_check_commit=land_check_commit)
```

Comment:
I think we still need to delete a few of these find_matching_merge_rules and stuff.

Reply:
Specifically, it's the one in merge() that I believe we need to remove. The others seem to cover slightly different scenarios